R/tbl_continuous.R

```diff
@@ -94,6 +95,7 @@ tbl_continuous <- function(data,
     digits = digits
   )
 
+  type <- assign_summary_type(data, variables = include, value = NULL)
```

Comment:
by restoring it

Reply:
type<- rep_named(x$inputs$include, list("categorical")) use something along these lines